SingularityService/src/main/java/com/hubspot/singularity/config/SingularityConfiguration.java

```diff
@@ -135,6 +135,10 @@
 
   private long zookeeperAsyncTimeout = 5000;
 
+  private int coreThreadpoolSize = 8;
```

Comment:
Sharing them allows to factor the thread pool management out of a lot of components which in turn no longer need to be managed (only managed thing was the threadpool shutdown). The "8" is a magic number based on the number of thread pools in existence. We have a metric which tracks the number of threads actually used and I may adjust later.